RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Open Jitsi call even if server isn't found

## Proposed changes
The purpose of this PR is to implement a new flow for Jitsi Meetings. 
Today, the mobile app can only open meeting rooms from known servers. When we tap a Jitsi URL or the app is opened from deep link, we use the Jitsi URL pattern to identify the server, connect to it and only then we open the call. 

A new flow was requested to open Jitsi meetings even we do not find the server.


## Issue(s)	
N/A

## How to test or reproduce
To test the new behaviour, will be necessary create a Jitsi meet with a random url, for example `https://jitsi.rocket.chat/test-call`, Send as a link on a chat, tap this link  and then a meeting room should open normally.

For a regression test, try opening some normal meeting rooms, for example:

* open a chat -> tap on top menu to open the action screen: 
  * try to make a voice call; 
  * try to make a video call;

everything needs works normally 

## Screenshots

## Before

https://user-images.githubusercontent.com/14894380/137550192-309b4bfa-9fcf-4192-9195-da792ec2e05e.mov


## After

https://user-images.githubusercontent.com/14894380/137550156-bda77c55-3cde-4212-8f9b-f56b3b5a275d.mov


## Types of changes




## Checklist



## Further comments
N/A
